java-analyzer-bundle.core/src/main/java/io/konveyor/tackle/core/internal/util/OpenSourceLibraryExclusionManager.java

```diff
@@ -0,0 +1,81 @@
+package io.konveyor.tackle.core.internal.util;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+import org.eclipse.jdt.core.IPackageFragmentRoot;
+import static org.eclipse.jdt.ls.core.internal.JavaLanguageServerPlugin.logInfo;
+
+public class OpenSourceLibraryExclusionManager {
+    // TODO (pgaikwad): set this file path at build time
+    private static final String MAVEN_INDEX_FILE = "/usr/local/etc/maven.default.index";
```

Comment:
Could this break running containerless?

Reply:
@eemcmullan where is this path supposed to be in containerless? It wouldn't break it, but the feature won't work. We still do the filtering post-search in analyzer-lsp. So results won't be affected. Its just that containerless wouldn't be able to take advantage of performance benefits from this